applications/ContactStructuralMechanicsApplication/symbolic_generation/ALM_frictional_mortar_condition/generate_frictional_mortar_condition.py

```diff
@@ -36,12 +36,12 @@ def real_norm(input):
     return output
 
 lhs_string = ""
```

Comment:
wouldn't it be better to use the multiline string ?

Reply:
Probably, I don't know about it